packages/components/drawer/src/utils.ts

```diff
@@ -0,0 +1,36 @@
+import { keyframes } from '@stitches/react'
```

Comment:
This introduces another method for styling/animating in our codebase. I would say we stick to Tailwind if possible. For the animation part, use the transition or animation class whenever you can. We will plan to have a proper animation library in the future.